corehq/apps/indicators/utils.py

```diff
@@ -1,10 +1,10 @@
 from couchdbkit import ResourceNotFound
 from django.conf import settings
-from corehq.util.quickcache import quickcache
+from corehq.util.quickcache import skippable_quickcache
 from dimagi.utils.couch.database import get_db
 
 
-@quickcache([], timeout=60)
+@skippable_quickcache([], lambda: settings.UNIT_TESTING, timeout=60)
```

Comment:
my point was to disable caching for tests on a function-by-function basis when we know it's ok to do so for that function, which is what i did in this pr. (rather than disabling caching for all functions in tests as you suggested)

but now that i think about this more, i think the right solution here is to always force the cache to be cleared when the function's source changes, in which case we should never disable caching during tests